src/content/docs/codestream/start-here/codestream-new-relic.mdx

```diff
@@ -0,0 +1,224 @@
+* `name` - example: `newrelic/beta-docs-site`
+```
+mutation {
+  referenceEntityCreateOrUpdateRepository(repositories: [{accountId: [YOUR_ACCOUNT_ID], name: "[REPO_NAME]", url: "[REPO_URL]"}]) {
+    created
+    failures {
+      guid
+      message
+      type
+    }
+  }
+}
+```
+In order to find back the entity you have created you can use a query like the following.  Note that the URL you
+provided to `referenceEntityCreateOrUpdateRepository` gets saved as an entity tag.
+```
+{
+  actor {
+    entitySearch(query: "name = 'a name' OR tags.url = 'a url'") {
+      count
+      query
+      results {
+        entities {
+          guid
+          name
+          tags {
+            key
+            values
+          }
+        }
+      }
+    }
+  }
+}
+
+```
+**Step 2: Associate the repository entity to your application entity**
+
+First find the GUID for the application you want to associate your repository too.  
+
+Parameters:
+* `sourceEntityGuid` - the entity GUID of the application
+* `targetEntityGuid` - the entity GUID of your Repository
+* `type` - always `BUILT_FROM`
+
+```
+mutation {
+  entityRelationshipUserDefinedCreateOrReplace(sourceEntityGuid: "", targetEntityGuid: "", type: BUILT_FROM) {
+    errors {
+      message
+      type
+    }
+  }
+}
+```
+
+To see all entities related to your Repository you can do a query like this:
+```
+{
+  actor {
+    entity(guid: "[YOUR_REPOSITORY_GUID]]") {
+      relatedEntities(filter: {direction: BOTH, relationshipTypes: {include: BUILT_FROM}}) {
+        results {
+          target {
+            entity {
+              name
+              guid
+              type
+            }
+          }
+          type
+        }
+      }
+      name
+      type
+      tags {
+        values
+        key
+      }
+    }
+  }
+}
+
+```
+
+**Step 3: Cleanup (if needed)**
+
+Delete a repository with the following query:
+```
+mutation DeleteRepository {
+  entityDelete(guids: "[ENTITY_GUID_HERE]]") {
+    deletedEntities
+    failures {
+      message
+      guid
+    }
+  }
+}
+
+```
+
+  </Collapser>
+
+</CollapserGroup>
+
+## Install APM agents with CodeStream [#install-agents]
+
+Requirements for installing New Relic APM agents via CodeStream:
+* [New Relic account](https://newrelic.com/signup)
+* [New Relic user key](https://one.newrelic.com/launcher/api-keys-ui.api-keys-launcher)
+* [CodeStream and New Relic connection](#connect)
+* A supported language application codebase
+    * .NET
+    * Java
+    * Node.JS
+
+With CodeStream connected to New Relic, when you're working on an application's codebase that's not being monitored by New Relic, CodeStream will offer to instrument that application for you.
+
+Like [New Relic's guided install](/docs/full-stack-observability/observe-everything/get-started/new-relic-guided-install-overview/), CodeStream will walk you through and automate all of the steps to installing the APM agent to start sending data to New Relic.
+
```

Comment:
Another feature will be discussing errors.  I am not sure if the docs should be added here or errors inbox.  It is just being finished now but I will ask Cindy if there are some mocks so you can see what it basically looks like.

The tl;dr is that it lets you discuss errors with other new relic users as well as your codestream teammates.  The comments a user puts on an Error Group in New Relic will be present when a user looks at that Error Group in CodeStream

Reply:
Is this self-evident in the UI? Or is there anything that needs to be done to let people do this? @obarney

Reply:
@paperclypse I was thinking mostly what Dave said but also thinking we should call out some of the non-intuitive features like that we have markdown support.  For example, you can preview the markdown by clicking 
<img width="83" alt="Screen Shot 2021-10-20 at 8 18 10 AM" src="https://user-images.githubusercontent.com/47154052/138123126-7afb2cc8-50ad-4aff-a8a7-178fdd4a3667.png">